### PR TITLE
Major UI Fixes

### PR DESCRIPTION
**Pull request by zrthxn**
Made a few UI fixes nothing major.
Conflicts arose but were quelled.

### CODE REVIEW

**Comment by zrthxn** [comment]
Yea some more conflicts were fixed

File: plasma_for_covid/settings.py [hunk 1]
```diff
@@ -81,12 +81,12 @@
 
 DATABASES = {
```
Comment (abbasidaniyal) [inline plasma_for_covid/settings.py:83]: Fix required

File: templates/base.html [hunk 2]
```diff
@@ -1,35 +1,77 @@
...
+		{{ form.media }}
+	{% endif %}
 </head>
 
 <body>
+	{% include 'header.html' %}
+	
+	{% block content %}
+	{% endblock %}
+
```
Comment (abbasidaniyal) [inline templates/base.html]: Messages should come before the block content

**Review by abbasidaniyal — CHANGES_REQUESTED** [review]

**Comment by abbasidaniyal** [comment]
Update local master with the project master. Rebase your local branch with master and push. Cleaned git flow rahega

**Comment by abbasidaniyal** [comment]
fixes #1

File: templates/base.html (hunk 2 above)
Comment (zrthxn) [reply templates/base.html]: Done

**Pull request merged**